deploy/olm-catalog/multicluster-operators-subscription/manifests/apps.open-cluster-management.io_gitopsclusters_crd_v1beta1.yaml

```diff
@@ -14,119 +12,6 @@ spec:
     singular: gitopscluster
   scope: Namespaced
   versions:
-  - name: v1alpha1
```

Comment:
I noticed that the gitopscluster CR used in the multicluster-integrations no longer has both versions v1alpha1 and v1beta1, only v1beta1. Any idea if v1alpha1 should still be intact?